src/firebase/auth.js

```diff
@@ -8,8 +8,17 @@ export const doCreateUserWithEmailAndPassword = async (email, password) => {
   return createUserWithEmailAndPassword(auth, email, password);
 };
 
-export const doSignInWithEmailAndPassword = (email, password) => {
-  return signInWithEmailAndPassword(auth, email, password);
+export const doSignInWithEmailAndPassword = async (email, password) => {
+  try {
```

Comment:
Small nit, but the try/catch statement is unnecessary if you plan on just throwing the error anyway.